### ISSUE TITLE
Support for XML format

### ISSUE THREAD

**Issue by matthesrieke** [matthesrieke/xml-json-reader#2]
The app needs support for the XML format. A XML file with measurements is present in the `src/main/resources/data` directory.

Implement a class (see e.g. the `CsvReader.java` as a rudimentary template) that supports the XML file and reads the measurements accordingly.

[JAXB](https://www.baeldung.com/jaxb) is the standard framework fro processing XML in Java. Due to the fact that it supports XML schemas and a lot more, It is more complex than e.g. Jackson for JSON. For the feature within this application, no schema support is required. Try to find a pragmatic lightweight solution.

**Comment by mohammedsaad** [comment]
>  For the feature within this application, no schema support is required. Try to find a pragmatic lightweight solution.
I think i haven't got the point very well. do you want us to use another solution than JAXB or to work it on JAXB?

**Comment by matthesrieke** [comment]
You are right, this might sound a little confusing. My idea was to use JAXB but do not use its sophisticated schema support (which involves additional pre-building steps) as the XML structure in this data file is not too complex. HTH

**Comment by matthesrieke** [comment]
A good example on how to do this would be: https://howtodoinjava.com/jaxb/jaxb-unmarshaller-example/
